README.md

```diff
@@ -1,31 +1,95 @@
+to the CORE Rule Engine can be easily managed using the publicly available WireCloud operational dashboard, or making
+use of any of the REST clients able to programmaticly use the Perseo's `Rule API`. These rules will identify patterns
+that will trigger actions with Orion to create or update entities, or with other different components or external
+systems, such as Web (HTTP), Email (SMTP) or SMS (SMPP) servers.
+
+## Installation
+
+The instructions to install Perseo can be found in the
+[Installation Guide](https://perseo.readthedocs.io/en/latest/setup.html)
+
+## Usage
+
+Information about how to use Perseo can be found in the
+[User & Programmers Manual](https://perseo.readthedocs.io/en/latest/example1.html)
+
+## API
+
+APIs and examples of their usage can be found [here](https://perseo.readthedocs.io/en/latest/api.html)
+
+## Testing
+
+For performing a basic end-to-end test, you can follow the detailed instructions
+[here](https://perseo.readthedocs.io/en/latest/test.html).
+
+## Quality Assurance
+
+This project is part of [FIWARE](https://fiware.org/) and has been rated as follows:
+
+-   **Version Tested:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Version&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.version&colorB=blue)
+-   **Documentation:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Completeness&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.docCompleteness&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Usability&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.docSoundness&colorB=blue)
+-   **Responsiveness:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Time%20to%20Respond&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.timeToCharge&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Time%20to%20Fix&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.timeToFix&colorB=blue)
+-   **FIWARE Testing:**
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Tests%20Passed&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.failureRate&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Scalability&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.scalability&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Performance&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.performance&colorB=blue)
+    ![ ](https://img.shields.io/badge/dynamic/json.svg?label=Stability&url=https://fiware.github.io/catalogue/json/perseo.json&query=$.stability&colorB=blue)
+
+## More Information
+
+-   Refer to the
+    [Esper Reference Documentation](http://esper.espertech.com/release-6.1.0/esper-reference/html/index.html) for info
+    on how to use EPL as a rule language.
 
 | :dart: [Roadmap](documentation/roadmap.md) |
```

Comment:
Instead of including the roadmap icon+link at this point (which is a bit ugly) it would be nicer to include the whole "toolbar" at the top. 

![imagen](https://user-images.githubusercontent.com/1534240/55390900-7884a800-5538-11e9-801f-acd424386beb.png)

Have a look for instance how Orion README.md does at https://github.com/telefonicaid/fiware-orion/blob/master/README.md

Reply:
OK. I have created the toolbar in the top of the Github README.md. Although the Esper documentation link is still in the 'More Information' section. Do you have any other solution?

Fixed in 206a1246dba30113ff260e4d3057e6974a8a9299